app/routes/questionnaire.py

```diff
@@ -393,14 +395,34 @@ def get_view_submitted_response(schema, questionnaire_store):
             questionnaire_store,
             flask_babel.get_locale().language,
         )
-
     except ViewSubmittedResponseNotEnabled:
         raise NotFound
 
-    return render_template(
-        template="view-submitted-response",
-        content=view_submitted_response.get_context(),
-        page_title=view_submitted_response.get_page_title(),
+    return view_submitted_response.get_rendered_html()
+
+
+@post_submission_blueprint.route("download-pdf", methods=["GET"])
+@with_questionnaire_store
+@with_schema
+def get_view_submitted_response_pdf(
```

Comment:
consider doc strings

Reply:
Have added doc string. We should have a wider convo with the type on the style guide for doc strings and when we should have them etc. The doc strings we have mostly conform to `Sphinx` format, but I personally prefer the Google's style guide and it not as visually dense. We should discuss the options in a tech session and put together or follow a style guide.